app/views/csv_preview/access_limited.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :title, "Access Forbidden - GOV.UK" %>
```

Comment:
Is there any interest in adding new page text directly to locale strings?

Reply:
Ooh, yes, I should probably do that.